docs/developer-guides/reproducible-builds.md

```diff
@@ -0,0 +1,76 @@
+# <img class="dcr-icon" src="/img/dcr-icons/Code.svg" /> Reproducible Builds
+
+---
+
+The process of building software from source can be described as
+**reproducible** (or **deterministic**) if it can be run multiple times, under
+different conditions, and always yield the same result when given the same
+input.
+The output of a truly deterministic build will be identical down to every
+individual bit, such that equality can be measured using MD5/SHA1 hashes.
```

Comment:
these are bad examples of a cryptographic hash in 2020. let's say SHA256 since that is what the manifest currently commits to.